bcda/auth/api_test.go

```diff
@@ -50,9 +51,9 @@ func (s *AuthAPITestSuite) TestAuthToken() {
 	clientID, clientSecret, accessToken := uuid.New(), uuid.New(), uuid.New()
 	mock := &auth.MockProvider{}
 	mock.On("MakeAccessToken", auth.Credentials{ClientID: clientID, ClientSecret: clientSecret}).
-		Return(accessToken, nil)
+		Return(accessToken, "", nil)
```

Comment:
this is the happy path scenario, right? if so can we mock the return as a value (ex. 1200) instead of empty string, and then assert it returned the correct value (vs asserting ExpiresIn returned empty)?